linux/LinuxProcessList.c

```diff
@@ -1433,7 +1433,7 @@ static bool LinuxProcessList_recurseProcTree(LinuxProcessList* this, openat_arg_
 
             uint64_t recheck = ((uint64_t)rand()) % 2048;
 
-            if (passedTimeInMs > 2000 || passedTimeInMs > recheck) {
+            if (passedTimeInMs > recheck) {
```

Comment:
This is intentional as refreshs of that metric are randomized otherwise and this ensures some freshnes.

Reply:
`recheck` is guaranteed to be at maximum `2047`.

Reply:
This slightly influences the distribution for the case where recheck has a value where the entry will now not be updated in a certain time window. This change is negligible, but that's why I asked what motivated this change now.